### ISSUE TITLE
Installation problem - libpng12-dev

### ISSUE THREAD

**Issue by carlschuurbiers** [wolfgangw/digital_cinema_tools_distribution#31]
I have the following problem when installing on a fresh installation of Ubuntu Studio 17.04 (Zesty Zapus):

```
sudo wget http://git.io/digital-cinema-tools-setup && bash digital-cinema-tools-setup
[sudo] password
URL transformed to HTTPS due to an HSTS policy
--2017-08-17 09:13:33--  https://git.io/digital-cinema-tools-setup
Resolving git.io (git.io)... 54.197.255.104, 54.204.29.243, 54.204.24.181, ...
Connecting to git.io (git.io)|54.197.255.104|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://raw.github.com/wolfgangw/digital_cinema_tools_distribution/master/digital-cinema-tools-setup [following]
--2017-08-17 09:13:34--  https://raw.github.com/wolfgangw/digital_cinema_tools_distribution/master/digital-cinema-tools-setup
Resolving raw.github.com (raw.github.com)... 151.101.112.133
Connecting to raw.github.com (raw.github.com)|151.101.112.133|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://raw.githubusercontent.com/wolfgangw/digital_cinema_tools_distribution/master/digital-cinema-tools-setup [following]
--2017-08-17 09:13:34--  https://raw.githubusercontent.com/wolfgangw/digital_cinema_tools_distribution/master/digital-cinema-tools-setup
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.112.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.112.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 28673 (28K) [text/plain]
Saving to: ‘digital-cinema-tools-setup.6’

digital-cinema-tools-setup.6        100%[=================================================================>]  28,00K  --.-KB/s    in 0,02s   

2017-08-17 09:13:34 (1,13 MB/s) - ‘digital-cinema-tools-setup.6’ saved [28673/28673]

digital-cinema-tools-setup v1.2015.02.20
OS: ubuntu
build-essential: OK
curl: Installation required
libxslt1-dev: Installation required
libxml2-dev: Installation required
libexpat1-dev: Installation required
xmlsec1: Installation required
libreadline-dev: Installation required
zlib1g: OK
zlib1g-dev: OK
libssl-dev: Installation required
imagemagick: OK
mplayer: OK
sox: OK
git-core: Installation required
libtiff-dev: Installation required
libpng12-dev: Installation required

digital-cinema-tools-setup: 10 packages missing
digital-cinema-tools-setup: curl libxslt1-dev libxml2-dev libexpat1-dev xmlsec1 libreadline-dev libssl-dev git-core libtiff-dev libpng12-dev

Please enter your password
Hit:1 http://archive.canonical.com/ubuntu zesty InRelease
Hit:2 http://security.ubuntu.com/ubuntu zesty-security InRelease
Hit:3 http://de.archive.ubuntu.com/ubuntu zesty InRelease
Hit:4 http://de.archive.ubuntu.com/ubuntu zesty-updates InRelease
Hit:5 http://de.archive.ubuntu.com/ubuntu zesty-backports InRelease
Reading package lists... Done                      
digital-cinema-tools-setup: Package preparation OK
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'libtiff5-dev' instead of 'libtiff-dev'
Package libpng12-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libpng12-dev' has no installation candidate
apt-get: Failed to install basic requirements
```

It seems to not able to find the 'libpng12-dev' package.
Any ideas?

Thanks,
Carl.

**Comment by brocky27** [comment]
I was able to get past this by adding the old package through the apt source list, see instructions on the page below. I'm still trying to install other dependencies though, wish me luck!

https://askubuntu.com/questions/840257/e-package-libpng12-0-has-no-installation-candidate-ubuntu-16-10-gnome

**Comment by wolfgangw** [comment]
Considered fixed with 39498769436e080c71d4eb4eaf04038b3d07f846.

Please remove the installer script and run ```wget http://git.io/digital-cinema-tools-setup && bash digital-cinema-tools-setup``` again.

**Comment by carlschuurbiers** [comment]
Yes!!
Successfully installed on Ubuntu Studio 14 on Oracle VM VirtualBox.
Many thanks!
